src/common/components/Latex/__tests__/Latex.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Latex from '../Latex';
+
+describe('Latex', () => {
+  it('renders only text', () => {
+    const wrapper = shallow((
+      <Latex>
+        This does not have latex in it
+      </Latex>
+    ));
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders text with LaTex in $...$', () => {
+    const formula = '2.8~\\mathrm{fb}^{-1}';
```

Comment:
I'd add a more real example from our existing records

Reply:
Actually records can have anything including this, there is no pattern here.

Reply:
yep but just to make sure the style is remaining as expected with bigger text